util/nbaUtils.js

```diff
@@ -10,3 +12,26 @@ module.exports.getTeamId = function (data, firstName, lastName) {
     })
     return returnValue
 }
+
+getCurrentMonday = function() {
+    let easternTime = moment_timezone().tz('America/New_York');
+    let monDate = (moment(easternTime).day("Monday"));
+  
+    return monDate;
+  }
+  
+getDaysOfWeek = function() {
+    let monDate = getCurrentMonday();
+    let daysOfWeek = [];
+
+    for(let i = 0; i < 7; ++i) {
+        let addDate = monDate;
+        daysOfWeek.push(addDate.format("YYYYMMDD"));
+        addDate = monDate.add(1, "days");
+    }
+
+    return daysOfWeek;
+}
+
+module.exports.getTeamId = getTeamId;
+module.exports.getDaysOfWeek = getDaysOfWeek;

```

Comment:
I like this. Makes it easy to see which functions this module has. Can we add these to the top of the file?

Reply:
i think its traditionally on the bottom of the file